### ISSUE TITLE
Python virtual environment not detected in the Select Interpreter option

### ISSUE THREAD

**Issue by divye1995** [microsoft/pylance-release#1194]
## Environment data

-   Language Server version: 2021.4.2
-   OS and version: OS X Catalina 10.15.7
-   Python version (& distribution if applicable, e.g. Anaconda): Python3

## Expected behaviour

I should be able to select virtual environment based python and be able to use auto-import for libraries added to my virtual env 

P.S Auto import and intellisense fails for these libs ( for example )
```
import requests_cache
from google.api_core import exception, retry
from google.cloud import pubsub_v1

```

## Actual behaviour

XXX
Unable to see the interpreter in options
![Screenshot 2021-04-26 at 8 49 48 PM](https://user-images.githubusercontent.com/10111323/116107790-f512f200-a6d0-11eb-8dba-dca6dc3effaa.png)
 

and  
Auto import or intellisense fails to work
![Screenshot 2021-04-26 at 8 50 31 PM](https://user-images.githubusercontent.com/10111323/116107908-0fe56680-a6d1-11eb-9a1d-11827e4cbdf0.png)

## Logs

Cant share as the paths for the project involve some names that i cant share , though I can say there are no errors 

Basically I m not able to use auto import for libraries installed in the virtualenv , which is causing really alot of problems as I have to switch to documentation instead of using the benefit of the intellisense or auto-import
